corehq/apps/translations/views.py

```diff
@@ -118,6 +119,30 @@ def download_bulk_multimedia_translations(request, domain, app_id):
     return export_response(temp, Format.XLS_2007, filename)
 
 
+# TODO: DRY up with upload_bulk_app_translations below
+@no_conflict_require_POST
+@require_can_edit_apps
+@get_file("bulk_upload_file")
+def upload_bulk_multimedia_translations(request, domain, app_id):
+    lang = request.POST.get('language')
+    app = get_app(domain, app_id)
+    workbook, msgs = get_app_translation_workbook(request.file)
+    if workbook:
```

Comment:
when is workbook empty?

Reply:
I don't know - this code is pre-existing.